application/classes/Ushahidi/Formatter/Post/CSV.php

```diff
@@ -144,25 +165,26 @@ private function writeStreamToFS($stream)
 
 	}
 
-	private function getValueFromRecord($record, $keyParam){
+	private function getValueFromRecord($record, $keyParam, $attributes){
```

Comment:
I'm having a lot of trouble following this method... can you write the story in comments? ie. literate style comments